### PR TITLE
feat: added venly wallet

### PR DESCRIPTION
**Pull request by prodesert22**
## Summary

- Added Venly wallet in supported wallets

### CODE REVIEW

File: src/connectors/Venly/index.ts [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+      if (_account) account = _account;
+    } catch (error) {
+      if ((error as any).code === 4001) {
+        throw new UserRejectedRequestError();
+      } else {
+        warning(false, 'eth_requestAccounts was unsuccessful, falling back to enable');
+      }
+    }
+
+    return { provider: this.web3Provider.provider, account };
```
Comment (SarjuHansaliya) [inline src/connectors/Venly/index.ts:93]: I think this should be only `this.web3Provider`
```suggestion
    return { provider: this.web3Provider, account };
```

File: src/connectors/Venly/index.ts [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+  }
+
+  public async getProvider() {
+    return this.web3Provider.provider;
```
Comment (SarjuHansaliya) [inline src/connectors/Venly/index.ts:97]:
```suggestion
    return this.web3Provider;
```

File: src/connectors/Venly/index.ts [hunk 3]
```diff
@@ -0,0 +1,135 @@
+import { ExternalProvider, Web3Provider } from '@ethersproject/providers';
+import { NoEthereumProviderError } from '@pangolindex/web3-react-injected-connector';
+import { Venly } from '@venly/web3-provider';
+import { AbstractConnector } from '@web3-react/abstract-connector';
+import { AbstractConnectorArguments, ConnectorUpdate } from '@web3-react/types';
+import warning from 'tiny-warning';
+import { UserRejectedRequestError } from '../DefiConnector';
+
+const venlyClientID = process.env.VENLY_ID ?? 'Testaccount';
```
Comment (SarjuHansaliya) [inline src/connectors/Venly/index.ts:9]: do we have VENLY_ID, so that I can set it into github action secrets

**Review by SarjuHansaliya — CHANGES_REQUESTED** [review]
Thanks @prodesert22, I think we need PR in interface also to handle Venly related changes to show which wallet is connected.

File: src/connectors/Venly/index.ts (hunk 3 above)
Comment (prodesert22) [reply src/connectors/Venly/index.ts:9]: Not yet, i asked abdullah to fill out the Venly form so we can get this id

**Pull request merged**